webapp/components/wpt-results.html

```diff
@@ -210,7 +210,12 @@
                 <th><test-run test-run="[[testRun]]"></test-run></th>
               </template>
               <template is="dom-if" if="[[diffShown]]">
-                <th><test-run test-run="[[diffRun]]"></test-run></th>
+                <th>
+                  <test-run test-run="[[diffRun]]"></test-run>
+                  <template is="dom-if" if="[[diffFilter]]">
+                    <paper-icon-button icon="filter-list" onclick="[[toggleDiffFilter]]"></paper-icon-button>
```

Comment:
This is great! One improvement would be to give this some state or button feel, so that one can tell that clicking it does something.